fsw/cfe-core/src/inc/cfe_es.h

```diff
@@ -176,7 +177,7 @@ static inline uint32 CFE_ES_ResourceID_FromInteger(unsigned long Value)
  * @return Execution status, see @ref CFEReturnCodes
  * @retval #CFE_SUCCESS                 @copybrief CFE_SUCCESS
  */
-int32 CFE_ES_AppID_ToIndex(uint32 AppID, uint32 *Idx);
+CFE_RC_t CFE_ES_AppID_ToIndex(uint32 AppID, uint32 *Idx);
```

Comment:
Can we use `CFE_ReturnCode_t` ? I like spelling out things as much as possible

Reply:
A lot of typing, and causes more line wrapping...I'm open though if that's the consensus.

Reply:
Ah, I use autocomplete on Atom. I assume this type would be used by app developers so I'd rather be explicit. If it's just internal to cFE then I feel slightly better about abbreviations